financial-templates-lib/ExpiringMultiPartyEventClient.js

```diff
@@ -0,0 +1,161 @@
+const { delay } = require("./delay");
+const { Logger } = require("./logger/Logger");
+const { LiquidationStatesEnum } = require("../common/Enums");
+
+// A thick client for getting information about an ExpiringMultiParty events.
+// This client is kept separate from the main ExpiringMultiPartyClient to keep
+// a clear separation of concerns and to limit the overhead from querying chain necessarily.
+// If no updateThreshold is specified then default to updating every 60 seconds.
+class ExpiringMultiPartyEventClient {
+  constructor(abi, web3, empAddress, updateThreshold = 60) {
+    this.updateThreshold = updateThreshold;
+    this.lastUpdateTimestamp;
```

Comment:
nit: what does this do?

Reply:
This defines the variable so it can be set later on. It follows this line implementation from the existing emp client [here](https://github.com/UMAprotocol/protocol/blob/master/financial-templates-lib/ExpiringMultiPartyClient.js#L10) where it is used to define when the last time the client was updated.

Reply:
Not sure if I mis understtood your question?

Reply:
I just didn't know what the purpose was. After this is called, is the variable `null` or `undefined`? I'm guessing it isn't `0`, right?